tests/Silverback.Integration.Tests/Messaging/Validation/ValidatorConsumerBehaviorTests.cs

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using Silverback.Diagnostics;
+using Silverback.Messaging.Broker;
+using Silverback.Messaging.Broker.Behaviors;
+using Silverback.Messaging.Messages;
+using Silverback.Messaging.Sequences;
+using Silverback.Messaging.Validation;
+using Silverback.Tests.Integration.TestTypes;
+using Silverback.Tests.Logging;
+using Silverback.Tests.Types;
+using Silverback.Tests.Types.Domain;
+using Xunit;
+
+namespace Silverback.Tests.Integration.Messaging.Validation
+{
+    public class ValidatorConsumerBehaviorTests
```

Comment:
You only test with invalid messages. Is it ok because you cover the success case elsewhere?

Reply:
Wow, I forgot them! 😰 
I added the happy case for the producer and consumer behavior: `HandleAsync_ValidMessage_NoLogAndNoException`